### PR TITLE
Adding view used to analyze PPI responses

### PR DESCRIPTION
**Pull request by danqrodney**

### CODE REVIEW

File: rest-api/alembic/versions/8b1281fb55ed_add_ppi_views.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = '8b1281fb55ed'
+down_revision = '8d12872e0b77'
+branch_labels = None
+depends_on = None
+
+_QUESTIONNAIRE_RESPONSE_ANSWER_VIEW_SQL = """
```
Comment (markfickett) [inline rest-api/alembic/versions/8b1281fb55ed_add_ppi_views.py]: Same comment as usual -- please add an overview comment. (Who will care if I change this, where/how is it used?)

Comment (danqrodney) [reply rest-api/alembic/versions/8b1281fb55ed_add_ppi_views.py]: Added comment.

File: rest-api/alembic/versions/8b1281fb55ed_add_ppi_views.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    LEFT OUTER JOIN hpo ON p.hpo_id = hpo.hpo_id
+    LEFT OUTER JOIN code ac ON qra.value_code_id = ac.code_id
+    WHERE (ps.email IS NULL OR ps.email NOT LIKE '%@example.com') AND
+           (hpo.name IS NULL OR hpo.name != 'TEST')
+"""
+def upgrade():
+  op.execute(_QUESTIONNAIRE_RESPONSE_ANSWER_VIEW_SQL)
+
+def downgrade():
+  pass
```
[the remark this answers is not in the document]
Comment (danqrodney) [reply rest-api/alembic/versions/8b1281fb55ed_add_ppi_views.py]: We have never used downgrade before; my impression is it only kinda-sorta works with Alembic-generated stuff.

Instead if we ever needed to roll back changes to the schema, we'd roll forward some drop statements.

So we could probably delete all the Alembic-generated downgrade code. (I just hadn't bothered.)

CREATE OR UPDATE is unnecessary; Alembic guarantees you won't run the same migration twice.

[the remark this answers is not in the document]
Comment (markfickett) [reply rest-api/alembic/versions/8b1281fb55ed_add_ppi_views.py]: I didn't realize downgrades mostly didn't work; good to know. Then, seems fine!

**Pull request merged**